src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -3,24 +3,42 @@
 import * as ecc from '@bitcoin-js/tiny-secp256k1-asmjs';
 import { BIP32Factory } from 'bip32';
 import { payments, networks } from 'bitcoinjs-lib';
-import type { IHdWalletAccount } from '@/types';
-import { MAXIMUM_ACCOUNTS_TO_DISCOVER } from '@/constants';
+import BigNumber from 'bignumber.js';
+import type { IHdWalletAccount, ICoin } from '@/types';
 import { BaseProtocolAdapter } from '@/protocols/BaseProtocolAdapter';
+import { MAXIMUM_ACCOUNTS_TO_DISCOVER, PROTOCOL_BITCOIN } from '@/constants';
 import {
   BTC_COINGECKO_COIN_ID,
+  BTC_CONTRACT_ID,
   BTC_SYMBOL,
   BTC_SYMBOL_SHORT,
 } from '@/protocols/bitcoin/config';
 
 export class BitcoinAdapter extends BaseProtocolAdapter {
   bip32 = BIP32Factory(ecc);
 
-  override getCoingeckoCoinId() {
+  override getCoinSymbol(getShort:boolean) {
+    return getShort ? BTC_SYMBOL_SHORT : BTC_SYMBOL;
```

Comment:
Both values are the same. So I think the argument for the function should not be required and for this protocol one value should be returned.